WTraining/app/services/user.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const { users } = require('../models');
```

Comment:
Remember that the models should be defined with the first letter in capital letter